wp-frontend-profile.php

```diff
@@ -43,7 +43,7 @@ function wfep_fs()
             $wfep_fs = fs_dynamic_init(
                 array(
                     'id'                  => '5837',
-                    'slug'                => 'wp-front-end-profile',
+                    'slug'                => 'wpfep',
```

Comment:
@frifrafry Please revert back slug as it would break the premium plugin.

Reply:
Will do, Search&Replace error.